examples/posenet/src/decodePose.js

```diff
@@ -0,0 +1,160 @@
+/**
```

Comment:
Please check the original file indent. The current file use both 2 spaces and 4 spaces and other indent issues.